README.md

```diff
@@ -43,6 +43,18 @@ yarn dev
 
 ## Build the Distributable
 
+> [!WARNING]
```

Comment:
We still need to document how to do this. Added a ticket here https://github.com/holochain/kitsune2/issues/200